modules/apps/segments/segments-web/src/main/resources/META-INF/resources/js/components/shared/ClayIcon.es.js

```diff
@@ -0,0 +1,37 @@
+import React, {Component} from 'react';
+import {PropTypes} from 'prop-types';
+import ThemeContext from '../../ThemeContext.es';
+import getCN from 'classnames';
+
+class ClayIcon extends Component {
+	static contextType = ThemeContext;
+
+	static propTypes = {
+		className: PropTypes.string,
+		iconName: PropTypes.string
+	};
+
+	render() {
+		const {className, iconName} = this.props;
+
+		const classes = getCN(
+			'lexicon-icon',
+			`lexicon-icon-${iconName}`,
+			{
+				[className]: className
+			}
+		);
+
+		return (
+			<svg
```

Comment:
In regards to my comment about about the svg and <img> tag. You can just re-use this component I believe.

Reply:
This component can only be used for lexicon icons since it hides the path to the lexicon's `spritemap` inside. Although I suppose I could add some boolean prop if it's like a lexicon icon or module-specific icon...